src/cljs/athens/views/pages/node_page.cljs

```diff
@@ -452,20 +480,31 @@
 
          [:> PageBody
 
+          (when (and (empty? properties)
+                     (empty? children))
+            [placeholder-block-el uid])
+
+          ;; Properties
+          [:div
+           (when (seq properties)
```

Comment:
maybe we should `(or (seq properties) (seq children))` to decide if block is collapsed

Reply:
Didn't quite get this, asked in DM. Merging without it to let others use this code during the weekend if they're messing around with main.